vimeo-networking/src/main/java/com/vimeo/networking/model/VideoFile.java

```diff
@@ -104,9 +173,54 @@ public boolean isVP6() {
         return getType() == MimeType.VP6;
     }
 
-    /** @return true if this VideoFile doesn't have an expired field or if the expires date is before the current date */
-    public boolean isExpired() {
-        // If expires is null, we should probably refresh the video object regardless [KV] 3/31/16
-        return expires == null || expires.before(new Date());
+    public int getWidth() {
+        return width;
+    }
+
+    public int getHeight() {
+        return height;
+    }
+
+    public long getSize() {
+        return size;
+    }
+
+    public int getFps() {
+        return fps;
+    }
+
+    @Nullable
+    public String getMd5() {
+        return md5;
+    }
+
+    @Nullable
+    public Date getCreatedTime() {
+        return createdTime;
+    }
+    // </editor-fold>
+
+    // -----------------------------------------------------------------------------------------------------
+    // Equals/Hashcode
+    // -----------------------------------------------------------------------------------------------------
+    // <editor-fold desc="Equals/Hashcode">
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) {
+            return true;
+        }
+        if (o == null || !(o instanceof VideoFile)) {
+            return false;
+        }
+
+        VideoFile that = (VideoFile) o;
+
+        return (this.getLink() != null && that.getLink() != null) && this.getLink().equals(that.getLink());
+    }
+
+    @Override
+    public int hashCode() {
+        return this.getLink() != null ? this.getLink().hashCode() : 0;
```

Comment:
"this.getLink()" accessor is unnecessary. could just use "link" to keep things terse